pkg/scheduler/plugins/volumebinding/binder.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 	storagev1 "k8s.io/api/storage/v1"
+	storagev1beta1 "k8s.io/api/storage/v1beta1"
```

Comment:
Both `v1` and `v1beta1` is reserved in references, would they disturbe each other?